module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,51 @@
+#include "sort.hpp"
+#include <forward_list>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& words)
+{
+    std::deque<std::string> sorted_words;
+    std::string temp_word;
+
+    for(std::forward_list<std::string>::iterator it = words.begin(); it != words.end(); it++)
+    {
+        sorted_words.push_back(*it);
+    }
```

Comment:
You can initialize deque like this:
std::deque<std::string> sorted_words{words.begin(),  words.end()};
No need for push_back for.

Reply:
+1